schedule/functional/toolchain_zypper.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: toolchain_zypper
```

Comment:
please rename this to toolchain instead

Reply:
okay, thanks!

Reply:
It should be renamed inside the file as well.

Reply:
sure, thanks!